components/tag/stories/template.js

```diff
@@ -0,0 +1,77 @@
+import { html } from "lit-html";
+import { classMap } from "lit-html/directives/class-map.js";
+// import { ifDefined } from "lit-html/directives/if-defined.js";
```

Comment:
Still needed?

Reply:
This one maybe not - I'd left it in because any properties added will need it and I thought it might make it easier for someone else to edit if the asset was right there for them and they didn't have to find out what it was called. I'm open to removing it though!

Reply:
We can keep it around for now! Thanks :-)